SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -167,6 +168,7 @@ const Worker_ComponentId GDK_KNOWN_ENTITY_TAG_COMPONENT_ID = 2007;
 const Worker_ComponentId TOMBSTONE_TAG_COMPONENT_ID = 2008;
 const Worker_ComponentId ROUTINGWORKER_TAG_COMPONENT_ID = 2009;
 const Worker_ComponentId LAST_EC_COMPONENT_ID = 2010;
+const Worker_ComponentId STRATEGYWORKER_TAG_COMPONENT_ID = 2011;
```

Comment:
I'm not exactly sure, but it seems that either `ROUTINGWORKER_TAG_COMPONENT_ID` should be larger than `LAST_EC_COMPONENT_ID`, or `STRATEGYWORKER_TAG_COMPONENT_ID` should be smaller than `LAST_EC_COMPONENT_ID`. @ImprobableNic can you please clarify?

Reply:
Looking at where it is used, it should be equal : 
SpatialConstants : 360
```
inline bool IsEntityCompletenessComponent(Worker_ComponentId ComponentId)
{
	return ComponentId >= SpatialConstants::FIRST_EC_COMPONENT_ID && ComponentId <= SpatialConstants::LAST_EC_COMPONENT_ID;
}
```

Reply:
Thanks, I've updated the value. It's obvious now that you pointed it out and I've read the names but in case anyone else misses it I've also added a note.